src/Components/Modules/Tagging/tagging.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import samplesize from 'lodash.samplesize'
+import { Tagging } from './tagging'
+import { COLORS } from 'Common/_mocks/colors'
+
+/* Tagging Story
+------------------------------------------------------- */
+export default class MockState extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      tags: samplesize(COLORS, 5),
```

Comment:
Hey @collectedmind I am setting up storyshots and it runs all the stories and creates snapshots. Each time we run though there is a new random sample of colors.

May we use COLORS.slice(0, 5) instead to keep our tests deterministic

Reply:
Certainly.  Feel free to update that as need be.  The main reason it is randomized was so we get a sampling of different tag lengths.

Reply:
I am planning on switching this to use slice and deprecate the lodash.samplesize in a subsequent PR @collectedmind